template/template.go

```diff
@@ -355,6 +355,10 @@ func (e *executer) resolveExpression(expr ast.Expression) (*ResolvedValue, error
 
 		switch v := expr.Operand.(type) {
 		default:
+			if f, ok := expr.Annotation.(ast.Function); ok {
+				return NewResolvedValue("{:" + f.Identifier.String() + "}")
```

Comment:
What about a helper function _newFallbackValue_? We return fall-back value in many places.

Reply:
Implemented `newFallbackValue` that does all the type checking and correct wrapping.